.github/workflows/default.yml

```diff
@@ -15,16 +15,16 @@ jobs:
         os: [ ubuntu-latest, windows-latest ]
     name: '[${{ matrix.os }}] build plugin'
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Set up Java
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           distribution: 'adopt'
           java-version: 17
 
       - name: Gradle wrapper validation
-        uses: gradle/wrapper-validation-action@v1
+        uses: gradle/wrapper-validation-action@859c33240bd026ce8d5f711f5adcc65c2f8eafc1 # https://github.com/gradle/wrapper-validation-action/issues/66
```

Comment:
Would this be fixed by advancing to node 16 for actions?

Reply:
AFAIU we (Github Actions users) don't have a control over the Node version used by Github Actions. Gtihub has [announced](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/) they'll switch the runtime at some point, and most Actions migrated already. 
If the Github advanced node version to 16, the `v1` would simply break our pipeline. The [commit](https://github.com/gradle/wrapper-validation-action/commit/859c33240bd026ce8d5f711f5adcc65c2f8eafc1) I linked here properly supports Node 16.

We could wait until they move the `v1` tag, but I wanted to remove all deprecation warnings already 👀 
I'll keep monitoring their repository and if `v1` will contain necessary changes I'll revert it back, for consistency